securedrop/tests/test_manage.py

```diff
@@ -1,10 +1,48 @@
 # -*- coding: utf-8 -*-
 
+from flask_testing import TestCase
 import manage
+import mock
+from StringIO import StringIO
+import sys
 import unittest
+import __builtin__
+
+import journalist
+import utils
 
 
 class TestManagePy(unittest.TestCase):
     def test_parse_args(self):
         # just test that the arg parser is stable
         manage.get_args()
+
+
+class TestManagementCommand(TestCase):
+    def create_app(self):
+        return journalist.app
+
+    def setUp(self):
+        utils.env.setup()
+
+    def tearDown(self):
+        utils.env.teardown()
+
+    @mock.patch("__builtin__.raw_input", return_value='N\n')
```

Comment:
Does just `'raw_input'`  not work? I thought everything in `__builtin__` is implicitly, unqualified-ly imported into each new namespace.

It seems like the `return_value` should not have the trailing `\n` (same goes for the line below). Otherwise wouldn't it be the case that every time an account was created from `manage.py` that the username would contain a newline character? I know it doesn't matter as far as our primary concern, which is ensuring this test is testing what it's supposed to be testing, but it does seem like a legitimate secondary concern to try to mock the return values of functions realistically.